### ISSUE TITLE
locale error on new shell

### ISSUE THREAD

**Issue by vanthiyathevan** [zeit/hyper#1206]
- [x] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: `4.8.13-1-ARCH`
- **Hyper.app version**: `1.0.0`
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: https://gist.github.com/vanthiyathevan/838ef268bd2d3bbdb0bbac17f2f28181
- **Relevent information from devtools** _(CMD+SHIFT+I on Mac OS, CTRL+SHIFT+I elsewhere)_: N/A
- **The issue is reproducible in vanilla Hyper.app**: N/A

## Issue

Hello, 
Thank you for the awesome app. I've installed it using `AUR` package. But every time I open it I see this messages. This shows up on every new console.

```
manpath: can't set the locale; make sure $LC_* and $LANG are correct
manpath: can't set the locale; make sure $LC_* and $LANG are correct
```
I have `zsh` as my default shell.

running `locale` on `gnome terminal` gives this
```sh
LANG=en_CA.UTF-8
LC_CTYPE=en_CA.UTF-8
LC_NUMERIC="en_CA.UTF-8"
LC_TIME="en_CA.UTF-8"
LC_COLLATE=en_CA.UTF-8
LC_MONETARY="en_CA.UTF-8"
LC_MESSAGES="en_CA.UTF-8"
LC_PAPER="en_CA.UTF-8"
LC_NAME="en_CA.UTF-8"
LC_ADDRESS="en_CA.UTF-8"
LC_TELEPHONE="en_CA.UTF-8"
LC_MEASUREMENT="en_CA.UTF-8"
LC_IDENTIFICATION="en_CA.UTF-8"
LC_ALL=
``` 
vs running `locale` on hyper
```sh
locale: Cannot set LC_CTYPE to default locale: No such file or directory
locale: Cannot set LC_MESSAGES to default locale: No such file or directory
locale: Cannot set LC_ALL to default locale: No such file or directory
LANG=en_GB.UTF-8
LC_CTYPE=en_GB.UTF-8
LC_NUMERIC="en_GB.UTF-8"
LC_TIME="en_GB.UTF-8"
LC_COLLATE=en_CA.UTF-8
LC_MONETARY="en_GB.UTF-8"
LC_MESSAGES="en_GB.UTF-8"
LC_PAPER="en_GB.UTF-8"
LC_NAME="en_GB.UTF-8"
LC_ADDRESS="en_GB.UTF-8"
LC_TELEPHONE="en_GB.UTF-8"
LC_MEASUREMENT="en_GB.UTF-8"
LC_IDENTIFICATION="en_GB.UTF-8"
LC_ALL=
```

Also when text is selected pressing `ctrl` un-select all selected.

**Comment by ppot** [comment]
@vanthiyathevan text is fix in #1218

**Comment by glats** [comment]
I have the same error. The only workaround I tried is run ```export LC_ALL=es_CL.UTF-8 ``` after start hyper

**Comment by rkmax** [comment]
Today I have installed most recent version 1.3.3 and just opening the hyper app have the same issue on Archlinux and using zsh as my default shell, in my case

running `locale` on gnome terminal gives this
```
LANG=es_CO.utf8
LC_CTYPE=es_CO.utf8
LC_NUMERIC="es_CO.utf8"
LC_TIME="es_CO.utf8"
LC_COLLATE="es_CO.utf8"
LC_MONETARY="es_CO.utf8"
LC_MESSAGES="es_CO.utf8"
LC_PAPER="es_CO.utf8"
LC_NAME="es_CO.utf8"
LC_ADDRESS="es_CO.utf8"
LC_TELEPHONE="es_CO.utf8"
LC_MEASUREMENT="es_CO.utf8"
LC_IDENTIFICATION="es_CO.utf8"
LC_ALL=
```
vs running `locale` on hyper

```
LANG=es_419.UTF-8
LC_CTYPE=es_419.UTF-8
LC_NUMERIC="es_419.UTF-8"
LC_TIME="es_419.UTF-8"
LC_COLLATE="es_419.UTF-8"
LC_MONETARY="es_419.UTF-8"
LC_MESSAGES="es_419.UTF-8"
LC_PAPER="es_419.UTF-8"
LC_NAME="es_419.UTF-8"
LC_ADDRESS="es_419.UTF-8"
LC_TELEPHONE="es_419.UTF-8"
LC_MEASUREMENT="es_419.UTF-8"
LC_IDENTIFICATION="es_419.UTF-8"
LC_ALL=
```

**Comment by gluons** [comment]
For me, I just set `LANG` in  `env` option in my `.hyper.js` file to the locale of my terminal.

**Comment by hatsuo** [comment]
- **OS version and name**: Linux Ubuntu 18.04 LTS
- **Hyper.app version**: 2.0.0
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: -- unmodified --
- **The issue is reproducible in vanilla Hyper.app**: Is Vanilla

----

Why was this issue closed?

I tested both the latest stable and the canary versions and have the problem in both cases.

Bypassing this issue by manipulating your OS's locales or by editing hyper's environment variables might work but shouldn't this be fixed in a better, general way as multiple people are affected by this?

In my case the command `locale` resolves to `LANG=de.UTF-8` in hyper but to `LANG=de_DE.UTF-8` anywhere else.
`/etc/default/locale` also tells me that it should be `de_DE.UTF-8`.

**Comment by chabou** [comment]
Fixed by #2913 merged in canary branch and will be shipped with our next canary release

**Comment by hatsuo** [comment]
Thank you for the clarification.
